### ISSUE TITLE
Ensure links are distinguished from surrounding text in a way that does not rely on color.

### ISSUE THREAD

**Issue by msftedad** [facebook/docusaurus#9169]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#issues).

### Prerequisites

- [X] I'm using the latest version of Docusaurus.
- [ ] I have tried the `npm run clear` or `yarn clear` command.
- [ ] I have tried `rm -rf node_modules yarn.lock package-lock.json` and re-installing packages.
- [X] I have tried creating a repro with https://new.docusaurus.io.
- [ ] I have read the console error message carefully (if applicable).

### Description

Ensure links are distinguished from surrounding text in a way that does not rely on color.
User is not able to differentiate between the normal text and links.
Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds.
Users with low vision will face difficulties if link has no styling (such as underline) to distinguish it from the surrounding text.

### Steps to reproduce

1. Open https://docusaurus.io/docs URL.
2. Navigate to link best docusaurus link present under Introduction and testimonials using tab key.
3. Verify whether links are distinguished from surrounding text in a way that does not rely on color or not.

### Expected behavior

Ensure that links are distinguished from surrounding text in a way that does not rely on color.
 Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds.

### Actual behavior

Links are distinguished from surrounding text in a way that does not rely on color. Color is the only way through which a link is identified. Underline or Chevron is not present for the links.
![MicrosoftTeams-image (35)](https://github.com/facebook/docusaurus/assets/93735775/52cac761-c38e-4936-b236-d8f22b3472e5)
![MicrosoftTeams-image (34)](https://github.com/facebook/docusaurus/assets/93735775/f3992864-7dc2-4296-b193-dea7083b71cf)

### Your environment

Public source code:
Public site URL: https://www.fast.design/
Docusaurus version used:
OS: Windows 11, Version: 22H2, Build: 22621.1992  
Microsoft Edge Version: 114.0.1823.86 (Official build) (64-bit)
Tool Used: Accessibility Insights for Web.

### Self-service

- [ ] I'd be willing to fix this bug myself.

**Comment by ishsarin** [comment]
Hi, if this issue is open could I contribute?

**Comment by homotechsual** [comment]
So the contrast issue is fixable already by you on this one and should have been fixed in your colour choices. I agree however that our default link styling should be adjusted to include `underline` by default. To that end: facebookincubator/infima/pull/290

**Comment by msftedad** [comment]
Hi @homotechsual, @slorber , Any update on this issue?

**Comment by Josh-Cena** [comment]
All discussion in https://github.com/facebookincubator/infima/pull/290

**Comment by msftedad** [comment]
We have closed our issue as fixed. So closing this issue.
